app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -0,0 +1,2 @@
+@import 'bootstrap/scss/bootstrap';
```

Comment:
App has an error that does not allow it to run so I could not test its features and layout. Kindly make sure the app is running without any errors.

![app_error](https://user-images.githubusercontent.com/35504149/197310406-db0c283f-7c29-4866-8afb-f178f5cca821.png)

Reply:
Hello there, I dont see the errors on my end, do you have any idea about fixing this pls? @SajjadAhmad14